### ISSUE TITLE
Unable to start default Interactive Shell

### ISSUE THREAD

**Issue by mjainQA** [Komodo/KomodoEdit#2496]
### Short Summary
Komodo is unable to start default interactive shell. 

### Steps to Reproduce
1. Close bottom pane by clicking x. 
2. Click on the drop-down arrow for Interactive Shell from the side toolbar.
3. Click on 'Start/find/Toggle...' option
4. Notice Interactive Shell does not open.
5. This is reproducible from tools -> Interactive Shell -> Start/find... as well.

### Platform Information

*Komodo Edit or IDE?* IDE
*Komodo Version?* 10.2.2
*Operating System (and version)?* macOS 10.9.5

### Additional Information
![image](https://cloud.githubusercontent.com/assets/26605593/25202764/5600e438-250b-11e7-9124-f9e5ccd4032a.png)

**Comment by Naatan** [comment]
Is Ruby installed on the machine this was tested on?

Please share your error log after reproducing the issue.

You can find your error log at Help > Troubleshooting > View Log File.

**Comment by mjainQA** [comment]
Yes, Ruby is installed. This is reproducible with any selected default language.


[pystderr.txt](https://github.com/Komodo/KomodoEdit/files/938735/pystderr.txt)

**Comment by Naatan** [comment]
Can you install 10.2.1 on the same machine and see if it works there? ie. test if it's a regression in 10.2.2.

**Comment by mjainQA** [comment]
Its reproducible on 10.2.1

**Comment by mitchell-as** [comment]
@mjainQA What version of Ruby do you have on OSX? Can you also debug Ruby? I cannot remember, but it's possible that the interactive shell and debugging are connected in some way.

**Comment by Defman21** [comment]
It doesn't work w/out byebug for me, so Mitchell is right. Though it's not indicated in any form and the interactive pane immediately closes for me.

**Comment by mjainQA** [comment]
@Naatan This is reproducible with any selected default language for e.g. Python.

ruby -version
ruby 2.0.0p481

**Comment by Naatan** [comment]
Can you reproduce this using `Help > Troubleshooting > Restart in Safe-mode` ?

Please verify that when restarting in safe-mode Komodo should be using the default UI (all your settings would be temporarily reset).

**Comment by Defman21** [comment]
I'm able to reproduce the issue with Ruby shell (don't have the byebug gem installed - immediately closes; otherwise works fine)

**Comment by mitchell-as** [comment]
I fail to see what the issue here is. You need the byebug gem installed to use the Ruby interactive shell. If you cannot run the interactive shell, then you cannot debug Ruby. If you can debug Ruby but cannot run the interactive shell, then there is a bug. I have not inferred the latter from the comments given so far.

**Comment by cgchoffman** [comment]
Personally I see this when I try to open the Interactive Shell for Python almost every time I try to use the Python Interactive shell for the first time.
